docs/contributing/index.rst

```diff
@@ -338,44 +559,84 @@ A good etiquette to take over is:
   new PR to the old one. The new PR should be created by pulling from the
   old one.
 
-.. _new_contributors:
+.. _coding-guidelines:
 
-Issues for New Contributors
-===========================
+Coding Style Guidelines
+=======================
 
-New contributors should look for the following tags when looking for issues.  We
-strongly recommend that new contributors tackle "easy" issues first: this helps
-the contributor become familiar with the contribution workflow, and for the core
-devs to become acquainted with the contributor; besides which, we frequently
-underestimate how easy an issue is to solve!
+The following are some guidelines on how new code should be written. Of
+course, there are special cases and there will be exceptions to these
+rules. However, following these rules when submitting new code makes
+the review easier so new code can be integrated in less time. Uniformly
+formatted code makes it easier to share code ownership.
 
-.. topic:: good first issue tag
+In addition to the PEP8 standards, geomstats follows the following
```

Comment:
capitalize Geomstats

Reply:
I actually started not capitalizing because these are existing docs. We will confirm with @lpereira95 and @ninamiolane when capitalization works and when its not relevant